ecosystem/indexer-grpc/indexer-grpc-table-info/src/tailer_service.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright © Aptos Foundation
```

Comment:
would it be better to put this file under storage?

Reply:
more consistent with the existing structure. when we want to refactor in future, it is easier to find them in one place.